tests/fsharp/Compiler/CodeGen/EmittedIL/DeterministicTests.fs

```diff
@@ -139,7 +139,7 @@ let test() =
 
```

Comment:
The body of the tests here is the de-facto **integration test suite** for the ref assembly MVID functionality.
The tests have been here since the ref assembly feature was implemented, most of them were ignore due to the issue this PR addresses.